framework/transport/ieee1905_transport/include/mapf/transport/ieee1905_transport.h

```diff
@@ -220,7 +221,7 @@ class Ieee1905Transport {
         bool operator()(const DeFragmentationKey &lhs, const DeFragmentationKey &rhs) const
         {
             // compare based on src address first then messageId
-            int srccmp = memcmp(lhs.src, rhs.src, ETH_ALEN);
+            int srccmp = memcmp(lhs.src.oct, rhs.src.oct, ETH_ALEN);
```

Comment:
Now you can use operator ==

Reply:
I prefer to do this as well in a separate PR since it is changing the logic.